lib/src/ffmpeg/ffmpeg_command.dart

```diff
@@ -44,26 +44,22 @@ class FfmpegCommand {
   final List<CliArg> args;
 
   /// The graph of filters that produce the final video.
-  final FilterGraph filterGraph;
+  final FilterGraph? filterGraph;
 
   /// The file path for the rendered video.
   final String outputFilepath;
 
   /// Converts this command to a series of CLI arguments, which can be
   /// passed to a `Process` for execution.
   List<String> toCli() {
-    if (filterGraph.chains.isEmpty) {
-      throw Exception('Filter graph doesn\'t have any filter chains. Can\'t create CLI command. If you want to make a'
-          ' direct copy of an asset, you\'ll need a different tool.');
-    }
-
     return [
       for (final input in inputs) ...input.args,
       for (final arg in args) ...[
         '-${arg.name}',
-        arg.value,
+        if (arg.value != null) arg.value!,
```

Comment:
Why would the value of an argument be null?

Reply:
When using flags such as -y or -shortest where there is no need for an argument, which was not possible before.

Reply:
Ok. In keeping with some other behaviors, would it make sense for the argument object to have a `toCli()` which makes this decision, itself?

E.g.:

```dart
for (final arg in args) ...[
  ...arg.toCli(),
]
```